### DISCUSSION TITLE
Update to Pt-BR at last

### DISCUSSION THREAD

**Discussion by ernandoal** [aclap-dev/video-downloadhelper#933] [Language & Translation]
Here follows attachment with update to Brazilian Portuguese for 6 itens missing
[pt_BR.txt](https://github.com/user-attachments/files/15991746/pt_BR.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
